src/web/core/src/main/java/org/geoserver/web/data/layergroup/AbstractLayerGroupPage.java

```diff
@@ -262,6 +268,10 @@ public void onSubmit() {
                     lg.getStyles().add(entry.getStyle());
                 }
 
+                // update not queryable flag
+                Boolean queryable = queryableCheckBox.getModelObject();
+                if (!queryable) lg.setNotQueryableEnabled(true); else lg.setNotQueryableEnabled(null);
```

Comment:
You should be able to directly bind the property once the layer group is property is called "queryable"